application/extensions/admin/BoxesWidget/assets/boxes-widget.js

```diff
@@ -0,0 +1,44 @@
+let cardHeaderRedirection = function () {
+    $(".box-widget-card-header, .box-widget-card-text").click(function (event) {
+        console.log()
```

Comment:
when hovering over box-widget-card-text not the entire area is turning the cursor into a pointer, either adjust what should be clickable or display a cursor for it depending oin the design